php/context/class-fieldmanager-context-submenu.php

```diff
@@ -239,6 +239,12 @@ protected function add_data( $data_id, $option_name, $option_value, $unique = fa
 	 * @return bool Option updated successfully.
 	 */
 	protected function update_data( $data_id, $option_name, $option_value, $option_prev_value = '' ) {
+		// Meta is always stored as a string, so if this is a scalar value, cast
+		// it as a string to ensure that `update_metadata()` is able to correctly
```

Comment:
Should this be `*_option()` instead of `_metadata`, and the earlier reference to "Meta is" be "Options are"?